test/intl402/Locale/prototype/textInfo/output-object-keys.js

```diff
@@ -0,0 +1,24 @@
+// Copyright 2021 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-intl.locale.prototype.textInfo
+description: >
+    Checks that the return value of Intl.Locale.prototype.textInfo is an Object
+    with the correct keys and properties.
+info: |
+  get Intl.Locale.prototype.textInfo
+  ...
+  7. Perform ! CreateDataPropertyOrThrow(info, "direction", dir).
+  features: [Intl.Locale,Intl.Locale-info]
+---*/
```

Comment:
you use "verifyProperty" and "compareArray" here so you need to add  a 
"includes: [propertyHelper.js, compareArray.js]"
line here